docker/pytorch-aarch64/cpu_info.sh

```diff
@@ -89,14 +95,30 @@ function set_target {
   echo "Setting target to $target"
 
   case $target in
-    neoverse-n1 )
+    neoverse | neoverse-n1 )
       cpu="neoverse-n1"
       tune="neoverse-n1"
       arch="armv8.2-a"
       blas_cpu="NEOVERSEN1"
       blas_ncores=64
       acl_arch="armv8.2-a"
     ;;
+    neoverse-v1 )
+      cpu="neoverse-v1"
+      tune="neoverse-v1"
+      arch="armv8.4-a"
+      blas_cpu="NEOVERSEV1"
+      blas_ncores=64
+      acl_arch="armv8.2-a"
+    ;;
+    neoverse-n2 )
+      cpu="neoverse-n2"
+      tune="neoverse-n2"
+      arch="armv8.5-a"
```

Comment:
Same request as on https://github.com/ARM-software/Tool-Solutions/pull/109, I think a comment about v9 support would be a good idea.

Reply:
addressed.